app/components/Sharing/PublicAccess.tsx

```diff
@@ -78,7 +78,7 @@ function PublicAccess({ document, share, sharedParent }: Props) {
 
         const val = ev.target.value;
         setUrlId(val);
-        if (val && !SHARE_URL_SLUG_REGEX.test(val)) {
+        if (val && !UrlHelper.SHARE_URL_SLUG_REGEX.test(val)) {
```

Comment:
I think we should expose some methods on `UrlHelper` next as there is a lot of repetition still, eg.

`UrlHelper.isShareUrl(val)` rather than `UrlHelper.SHARE_URL_SLUG_REGEX.test(val)`